iothub_client/src/iothubtransport_amqp_connection.c

```diff
@@ -146,17 +146,17 @@ static void on_connection_state_changed(void* context, CONNECTION_STATE new_conn
 
 static void on_cbs_open_complete(void* context, CBS_OPEN_COMPLETE_RESULT open_complete_result)
 {
-    (void)context;
     (void)open_complete_result;
     if (open_complete_result != CBS_OPEN_OK)
     {
         LogError("CBS open failed");
+		update_state((AMQP_CONNECTION_INSTANCE*)context, AMQP_CONNECTION_STATE_ERROR);
```

Comment:
Please keep the indentation, and use spaces instead of tabs.

Reply:
I am using tabs and indentation looks fine in my editor, not sure why it looks off here. I will double check